forum/urls.py

```diff
@@ -1,16 +1,12 @@
-from django.conf.urls import url 
+from django.urls import path 
 from . import views 
 
 app_name = "forum"
 urlpatterns = [
-    # urls shared by everyone with index == login
-    url(r'^$', views.views, name="index"), 
-    url(r'^post$', views.post, name="post"), 
-    url(r'^logout$', views.logout, name="logout"), 
+    # Log_Reg page removed 
+    path("", views.HomePageView.as_view(), name="home"), 
+    path("user", views.UserView.as_view(), name="user"),
+    path("comment/<int:pk>", views.CommentDetailView.as_view(), name="comment_detail"),
```

Comment:
`<int:pk>` isn't going to work. That will only work if the model primary key is an integer. If you check the `Comment`, you'll notice we're using a `UUIDField` as the primary key. You'll need to change this to something else.

Reply:
Hey you are exactly right about UUID.  I didnt realize that an algorithm would be used to create an id.  Its much different than simply making an id unique.  Would like to see your thoughts on Django approach to passing UUID through url.  The first user info could be passed as a string I imagine as i can get() the user if i already know the id (e.g. eb0af0ce6347436690885cf3cb4566c7).

Reply:
[You can just use `uuid`](https://docs.djangoproject.com/en/2.0/topics/http/urls/#path-converters)